mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMapperProvider.java

```diff
@@ -57,28 +57,30 @@
 public class CassandraMapperProvider implements MapperProvider {
 
     private static final Factory MESSAGE_ID_FACTORY = new CassandraMessageId.Factory();
-    private static final MockMailboxSession MAILBOX_SESSION = new MockMailboxSession("benwa");
-    private static final CassandraCluster cassandra = CassandraCluster.create(new CassandraModuleComposite(
-        new CassandraAclModule(),
-        new CassandraMailboxModule(),
-        new CassandraMessageModule(),
-        new CassandraMailboxCounterModule(),
-        new CassandraMailboxRecentsModule(),
-        new CassandraModSeqModule(),
-        new CassandraUidModule(),
-        new CassandraAttachmentModule(),
-        new CassandraAnnotationModule(),
-        new CassandraFirstUnseenModule(),
-        new CassandraApplicableFlagsModule(),
-        new CassandraDeletedMessageModule()));
     public static final int MAX_ACL_RETRY = 10;
 
+    private final CassandraCluster cassandra;
     private final MessageUidProvider messageUidProvider;
     private final CassandraModSeqProvider cassandraModSeqProvider;
+    private final MockMailboxSession mailboxSession = new MockMailboxSession("benwa");
```

Comment:
good catch them! Can you then fix the other misusages please? Hint: git grep -i 'static.*mailboxSession'